cli/azd/resources/alpha_features.yaml

```diff
@@ -2,6 +2,8 @@
   description: "Support infrastructure deployments at resource group scope."
 - id: infraSynth
   description: "Enable the `infra synth` command to write generated infrastructure to disk."
+- id: aspire.autoConfigureDataProtection
```

Comment:
Just wondering, when would the feature be supported in all regions?

Reply:
My understanding is that the hope is to promote it from the EUAP regions into all regions with the next month or so.  Part of the goal of being able to do this earlier on our end is that it allows the aspire folks to test end to ends in EUAP.

So hopefully we can make this always true in a few months and not have this point in time wart.